contrib/coreos/user-data.example

```diff
@@ -76,6 +76,28 @@ coreos:
       Type=oneshot
       ExecStartPre=/usr/sbin/modprobe nf_conntrack
       ExecStart=/bin/sh -c "sysctl -w net.netfilter.nf_conntrack_max=262144"
+  - name: sync-clock-hourly.timer
+    command: start
+    content: |
+      [Unit]
+      Description=sync clock timer
+
+      [Timer]
+      OnCalendar=hourly
+      Unit=sync-clock-hourly.service
+
+      [Install]
+      WantedBy=multi-user.target
+  - name: sync-clock-hourly.service
+    command: enable
+    content: |
+
+      [Unit]
+      Description=sync clock service
+
+      [Service]
+      Type=simple
+      ExecStart=/bin/bash -c "/usr/sbin/ntpdate -b -u 0.pool.ntp.org"
```

Comment:
I think we should use the coreos ntp server group here https://github.com/coreos/coreos-overlay/blob/471cd901c4aa8987d24db4e392d582fb212c1f52/net-misc/ntp/files/ntp.conf#L2-L5 Is there a way to not specify a server directly and use the ntp config shipped with coreos instead?